src/sentry/utils/auth.py

```diff
@@ -448,3 +448,7 @@ def construct_link_with_query(path: str, query_params: dict[str, str]) -> str:
     if query_string:
         redirect_uri += f"?{query_string}"
     return redirect_uri
+
+
+class AuthenticatedHttpRequest(HttpRequest):
```

Comment:
wanna delete the one in api/base.py in favor of this one? although i think we use the DRF `Request` object, so may want to subclass that instead of `HttpRequest`

Reply:
I was looking for this. Thank you @JoshFerge. Updated and fixed.